app/src/main/java/cz/muni/fi/pv256/movio2/uco_422476/MainActivity.java

```diff
@@ -1,13 +1,58 @@
 package cz.muni.fi.pv256.movio2.uco_422476;
 
+import android.content.Intent;
+import android.support.v4.app.FragmentManager;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
-public class MainActivity extends AppCompatActivity {
+import java.util.ArrayList;
+import java.util.Calendar;
+
+public class MainActivity extends AppCompatActivity implements MainFragment.OnFilmSelectListener {
+
+    private boolean mTwoPane;
+    private ArrayList<Film> mFilmList = new ArrayList<>();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        mFilmList.add(new Film(Calendar.getInstance().getTime().getTime(), "", "Film 1", "", 1));
```

Comment:
preco je tu dvakrat za sebou getTime() ?

Reply:
Calendar.getInstance().getTime() vracia premennu typu Date, na ktorej getTime() vrati long. Ono to bolo aj tak len docasne :)